packages/user-facing-text/api.d.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Build a message from given template and arguments.
```

Comment:
This is a **public** api, thus we are more **strict** in the documentation.
Please add a small usage example, nobody would understand how the arguments/placeholders 
should be marked without said example.